.env.example

```diff
@@ -91,13 +91,21 @@ export BOOKWAREHOUSE_NAMESPACE=bookwarehouse
 # export ENABLE_EGRESS=true
 
 # optional: ENABLE_GRAFANA (true/false)
-# Default: false
+# Default: true
 # export ENABLE_GRAFANA=true
 
 # optional: ENABLE_FLUENTBIT (true/false)
 # Default: false
 # export ENABLE_FLUENTBIT=true
 
+# optional: ENABLE_PROMETHEUS (true/false)
+# Default: true
+# export ENABLE_PROMETHEUS=true
+
+# optional: ENABLE_PROMETHEUS_SCRAPING (true/false)
+# Default: false
```

Comment:
```suggestion
# Default: true
```

Reply:
These defaults are for the demo script, not the osm cli itself